src/SplitPane.js

```diff
@@ -330,14 +334,12 @@ class SplitPane extends React.Component {
           key="resizer"
           resizerClassName={resizerClassNamesIncludingDefault}
           split={split}
-          style={resizerStyle || {}}
+          style={resizerStyle || emptyObject}
```

Comment:
I wonder if theres a cleaner way for us to do this. What if we set default props in `Pane`?

Reply:
For what it's worth, I use the same pattern used here in my projects to prevent rerendering. 
Default props would work, I think; however, are you gaining anything from it as opposed to this approach?